app/vue/src/server/iframe.html.js

```diff
@@ -56,7 +56,8 @@ export default function({ assets, publicPath, headHtml }) {
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <script>
           if (window.parent !== window) {
-            window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = window.parent.__REACT_DEVTOOLS_GLOBAL_HOOK__;
+            window.__VUE_DEVTOOLS_GLOBAL_HOOK__ = window.parent.__VUE_DEVTOOLS_GLOBAL_HOOK__;
+            window.parent.__VUE_DEVTOOLS_CONTEXT__ = window.document;
```

Comment:
We don't unless we want to ensure Devtools looks directly at this context. It would possibly speed up start up time a bit. I can remove it if you want me to